src/runtime/audience-action-local-flows-test.js

```diff
@@ -795,15 +806,23 @@ describes.realWin('AudienceActionLocalFlow', (env) => {
         expect(form).to.not.be.null;
         expect(form.innerHTML).contains('newsletter_code_snippet');
         form.dispatchEvent(new SubmitEvent('submit'));
-        await tick();
+        await tick(3);
 
-        expect(env.win.fetch).to.be.calledTwice;
-        expect(env.win.fetch).to.be.calledWith(
-          'https://news.google.com/swg/_/api/v1/publication/pub1/completeaudienceaction?sut=abc&configurationId=newsletter_config_id&audienceActionType=TYPE_NEWSLETTER_SIGNUP'
+        const updatedWrapper = env.win.document.querySelector(
+          '.audience-action-local-wrapper'
         );
+        expect(updatedWrapper).to.be.null;
```

Comment:
The new assertions added are not reflected in the test case description ('submit event triggers completion event'). Can you either break this into smaller test cases that are each testing one behavior, or update the description to make it more general?

Reply:
Break them into different tests!